CRM/Core/Form.php

```diff
@@ -413,6 +413,9 @@ public function &add(
         }
       }
     }
+    $optionContext = $extra['option_context'] ?? NULL;
+    unset($extra['option_context']);
```

Comment:
@jitendrapurohit why are we unsetting `$extra['option_context'] `?

Reply:
@yashodha `option_context` is in array format which is causing all the problems here. The main aim of this code seems to assign `data-option-edit-context` in the json format. This is done in the later part of the code. This PR avoids $element to have an unnecessary array attribute `option_context` which leads to the warning message on the relationship form.